lib/atomic-management.js

```diff
@@ -419,10 +419,8 @@ export default {
    * configuration.
    */
   restoreGlobalConfig() {
-    atom.config.resetProjectSettings(
-      {},
-      null
-    );
+    atom.config.resetProjectSettings ({}, true)
+    atom.config.clearProjectSettings();
```

Comment:
Judging from the code linked below, clearProjectSettings is essentially just `this.resetProjectSettings({}, null)`. I'm fine with just removing the call to `atom.config.resetProjectSettings ({}, true)` and merging this in though.

https://github.com/atom/atom/blob/a1b03404e81fac37bebeb77276742730a388174c/src/config.js#L991

Reply:
Never mind. I see that calling `atom.config.resetProjectSettings ({}, true)` will trigger the call to `_resetSettings`. All good to merge this in I think.